src/probestack.rs

```diff
@@ -93,7 +107,13 @@ pub unsafe extern "C" fn __rust_probestack() {
         add    %rax,%rsp
 
         leave
+        .cfi_def_cfa rsp, 8
```

Comment:
Again this could be `.cfi_def_cfa_register %rsp`;`.cfi_adjust_cfa_offset -8` to exactly match the relative effect of the instruction rather than encoding knowledge of the absolute state.